sys/include/net/eui48.h

```diff
@@ -36,6 +36,49 @@ typedef struct {
     uint8_t uint8[6];            /**< split into 6 8-bit words. */
 } eui48_t;
 
+/**
+ * @name EUI-48 bit flags contained in the first octet
+ *
+ * @see IEEE 802-2001 section 9.2
+ * @{
+ */
+
+/**
+ * @brief Locally administered address.
+ */
+#define EUI48_LOCAL_FLAG    0x02
+
+/**
+ * @brief Group type address.
+ */
+#define EUI48_GROUP_FLAG    0x01
+/** @} */
+
+/**
+ * @brief Set the locally administrated bit in the EUI-48 address.
+ *
+ * @see IEEE 802-2001 section 9.2
+ *
+ * @param   addr    ethernet address
+ */
+static inline void eui48_set_local(eui48_t *addr)
+{
+    addr->uint8[0] |= EUI48_LOCAL_FLAG;
+}
+
+/**
+ * @brief Clear the group address bit to signal the address as individual
+ * address
+ *
+ * @see IEEE 802-2001 section 9.2
+ *
+ * @param   addr    ethernet address
+ */
+static inline void eui48_clear_group(eui48_t *addr)
+{
+    addr->uint8[0] &= ~EUI48_GROUP_FLAG;
+}
+
```

Comment:
Could you split out the code reordering into a separate commit?

Reply:
Ok